### PR TITLE
RBG v0.5.1 Bug Fixes

### PR DESCRIPTION
**Pull request by sdenom01**
**Story:** [When creating a TimelineEvent on an empty timeline, if you then attempt to update the timeline event, it will CREATE a new one](https://trello.com/c/5iCKj3ke/99-when-creating-a-timelineevent-on-an-empty-timeline-if-you-then-attempt-to-update-the-timeline-event-it-will-create-a-new-one)

**Story:** [“Let’s build together!” button should link to registration page](https://trello.com/c/g0MrNdCo/100-lets-build-together-button-should-link-to-registration-page)

**Story:** [GrowTimelineEdit WYSIWYG Editor is applying really weird spacing to content that has a return character in it](https://trello.com/c/C5rrVU5z/116-growtimelineedit-wysiwyg-editor-is-applying-really-weird-spacing-to-content-that-has-a-return-character-in-it)

**Description:** Aims to fix up any of the v0.5.1 bugs that have been reported, any future bug fixes that pertain to v0.5.1 should be branched from this branch.

### CODE REVIEW

File: src/GrowTimeline/GrowTimeline.jsx [hunk 1]
```diff
@@ -83,11 +83,14 @@ export default class GrowDetails extends React.Component {
         });
 
         growService.deleteTimelineEvent(growId, event._id).then(event => {
-                var events = (foundEventIndex !== -1) ? this.state.events.splice(foundEventIndex, 1) : this.state.events;
-
-                this.setState({
-                    events: events
-                })
+                // var events = (foundEventIndex !== -1) ? this.state.events.splice(foundEventIndex, 1) : this.state.events;
```
Comment (saltMonger) [inline src/GrowTimeline/GrowTimeline.jsx:86]: Are these commented snippets of code necessary for future use?

File: src/GrowTimeline/GrowTimeline.jsx [hunk 2]
```diff
@@ -83,11 +83,14 @@ export default class GrowDetails extends React.Component {
...
+                //
+                // this.setState({
+                //     events: events
+                // })
+
+                // TODO: Ugly AF, need to do this without refreshing
+                window.location.reload();
```
Comment (saltMonger) [inline src/GrowTimeline/GrowTimeline.jsx:93]: We should probably think of the correct way to get these to refresh instead of just doing a window reload.

**Review by saltMonger — CHANGES_REQUESTED** [review]

File: src/GrowTimeline/GrowTimeline.jsx (hunk 1 above)
Comment (sdenom01) [reply src/GrowTimeline/GrowTimeline.jsx:86]: Potentially useful for non-refresh changes that we need to make...

File: src/GrowTimeline/GrowTimeline.jsx (hunk 2 above)
Comment (sdenom01) [reply src/GrowTimeline/GrowTimeline.jsx:93]: BLAH..... you're right.

**Pull request merged**